### ISSUE TITLE
Accessibility improvements

### ISSUE THREAD

**Issue by RafaPolit** [huridocs/uwazi#2910]
- [ ] Use full language names instead of two-letter abbreviations.  
- [ ] If above not possible, use ARIA tittles to include the full name of the language, as screen readers read: "EN", "ES", "PT", which makes no sense
- [ ] Overhaul review that screen readers are reading primary pages correctly for most of the content.

**Comment by simonfossom** [comment]
New iteration of the design: https://www.figma.com/file/xjsz7r65epN4B0b96zilac/UWAZI-redesigns?node-id=133%3A0

Let me know if you have any new considerations.

**Comment by RafaPolit** [comment]
Lets focus on the first two action points for now.

**Comment by nwyu** [comment]
@RafaPolit I misspoke yesterday. If we must prioritize, the focus should be on the screen reader (the third point)! The first two related to language updates would be a nice-to-have.

**Comment by RafaPolit** [comment]
We are going to focus exclusively on the languages section for now and ensure it's:
- Among the first things read (not necessarily visually in a different place)
- That it reads the full language name (not necessarily show the full language for now)

**Comment by simonfossom** [comment]
Check this: https://stackoverflow.com/questions/54124431/how-can-i-use-partly-another-language-in-html5-aria-label
